gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -234,13 +234,6 @@ add_setup_done_file (void)
 static void
 done_cb (GtkButton *button, GisSummaryPage *page)
 {
-  gchar *file;
```

Comment:
@cosimoc I am a bit confused. This commits removes:
```
  gchar *file;

  /* the tour is triggered by $XDG_CONFIG_HOME/run-welcome-tour */
  file = g_build_filename (g_get_user_config_dir (), "run-welcome-tour", NULL);
  g_file_set_contents (file, "yes", -1, NULL);
  g_free (file);
```
So your request should be already satisfied. If no, which is the code you are talking about?
the commits hash is not available anymore but the commit title is `welcome-tour: adapt for endless`